grader/flags.py

```diff
@@ -0,0 +1,83 @@
+from flagset import Flag, FlagSet
+
+fset = FlagSet({
+    "token": Flag(
```

Comment:
Question: How do you mark this as a positional argument in FlagSet? If the `cmdline_name` does not start with `--`?

Reply:
ah yes. the conventions are similar to argparse